CHANGELOG.rst

```diff
@@ -37,6 +37,8 @@ Current Developments
   ``--cache-everything`` when starting xonsh.
 * Added a workaround to allow ctrl-c to interrupt reverse incremental search in
   the readline shell
+* When a subprocess exits with a signal (e.g. SIGSEGV), a message is printed,
+  similar to Bash.
```

Comment:
Haha whoops, can you merge from master and then move this bit to the new Current Developments section, please? Sorry for the run around, we did the v0.3.0 release in between when you last pulled or forked and now.